rest/route/github.go

```diff
@@ -223,6 +223,18 @@ func (gh *githubHookApi) Run(ctx context.Context) gimlet.Responder {
 				return gimlet.MakeJSONInternalErrorResponder(errors.Wrap(err, "removing hook"))
 			}
 		}
+
+	case *github.MergeGroupEvent:
```

Comment:
Is this meant to be temporary? If yes, can we please add a ticket number or comment indicating what it's doing and a todo to remove it?

Reply:
In that case, I would suggest adding a comment similar to [this format](https://github.com/malikchaya2/evergreen/blob/46deb4bceb4c3d745bc6250a60d04867851c08ed/rest/route/project.go#L477-L479).